object-cache.php

```diff
@@ -328,12 +328,6 @@ function get_multi( $groups ) {
 					$return[ $key ] = $mc->get( $key );
 				}
 			}
-
-			if ( $to_get ) {
```

Comment:
This change has since been merged via https://github.com/Automattic/wp-memcached/pull/30 instead.